nano/rpc_test/rpc.cpp

```diff
@@ -4426,73 +4426,96 @@ TEST (rpc, account_info)
 	nano::system system (24000, 1);
 	nano::keypair key;
 	nano::genesis genesis;
+
+	auto & node1 (*system.nodes[0]);
+	enable_ipc_transport_tcp (node1.config.ipc_config.transport_tcp);
+	nano::node_rpc_config node_rpc_config;
+	nano::ipc::ipc_server ipc_server (node1, node_rpc_config);
+	nano::rpc_config rpc_config (true);
+	nano::ipc_rpc_processor ipc_rpc_processor (system.io_ctx, rpc_config);
+	nano::rpc rpc (system.io_ctx, rpc_config, ipc_rpc_processor);
+	rpc.start ();
+
+	boost::property_tree::ptree request;
+	request.put ("action", "account_info");
+	request.put ("account", nano::account ().to_account ());
+
+	// Test for a non existing account
+	{
+		test_response response (request, rpc.config.port, system.io_ctx);
+		system.deadline_set (5s);
+		while (response.status == 0)
+		{
+			ASSERT_NO_ERROR (system.poll ());
+		}
+
+		auto error (response.json.get_optional<std::string> ("error"));
+		ASSERT_TRUE (error.is_initialized ());
+		ASSERT_EQ (error.get (), "Account not found");
```

Comment:
You should be able to use the error code rather than hardcode the string, e.g:
```
std::error_code ec (nano::error_common::account_not_found);
ASSERT_EQ (*error, ec.message ());
```

Reply:
Thanks, went ahead and changed other cases of hardcoded strings in `rpc.cpp`